### PR TITLE
Fix multiline issue

### PR DESCRIPTION
**Pull request by FranciscoG**
QueUp added this new feature recently that Dub+ was messing with. They added the ability to add multiline chats by changing the chat input to a `textarea` and allowing `shift + enter` to start a new line while pressing just `enter` sends the chat. But when we turned on autocomplete, pressing enter just entered a new line and we were forced to click on the arrow to send the text

I also replace node-sass with sass because I was having issues compiling

### CODE REVIEW

**Review by coryshaw1 — APPROVED** [review]

**Comment by FranciscoG** [comment]
just doing some last minute testing before merge and something is off.  I technically _works_ but when you do `shift+enter` it inserts 2 lines instead of 1.  Going to mess around with it more to see if I can figure it out

File: src/js/modules/autocomplete.js [hunk 1]
```diff
@@ -157,68 +175,69 @@ var chatInputKeyupFunc = function(e){
...
-    KEYS.enter, 
-    KEYS.up, 
-    KEYS.down,
-    KEYS.left,
-    KEYS.right
-  ], 
+var chatInputKeydownFunc = function (e) {
+  var emptyPreview = !document.querySelector("#autocomplete-preview > li");
+  var isValidKey = _.includes(
+    [KEYS.tab, KEYS.enter, KEYS.up, KEYS.down, KEYS.left, KEYS.right],
```
Comment (coryshaw1) [inline src/js/modules/autocomplete.js:209]: Could we be sending enter twice here? Shift + Enter handled once by Queueup, and then an extra enter handled by Dub+?

**Comment by FranciscoG** [comment]
Got it working finally!

chat

File: src/js/modules/autocomplete.js (hunk 1 above)
Comment (FranciscoG) [reply src/js/modules/autocomplete.js]: it's something in their code. I see them manually inserting the `\n` on their end

**Pull request merged**